src/components/AssignedUsers/hooks/useAssignedUsers/useAssignedUsers.js

```diff
@@ -79,7 +79,7 @@ const useAssignedUsers = ({ grantedToIds = [], permissionSetId, tenantId }, opti
       }));
     },
     {
-      enabled: Boolean(grantedToIds.length && permissionSetId),
+      enabled: Boolean(permissionSetId),
```

Comment:
if you enable query when array is empty, do no load data in querFn when array is empty - there is no sense to load nothing

Reply:
Agree on that.